tests/app/api/test_workflows/graph_to_asset_hierarchy_workflow.yml

```diff
@@ -5347,13 +5347,13 @@ assets:
     IdentifiedObject.mRID: f17695aa-9aeb-11e5-91da-b8763fd99c5f
     IdentifiedObject.name: FORSMARK
     Substation.Region: f17695af-9aeb-11e5-91da-b8763fd99c5f
-    Substation.Terminal: 2dd901c6-bdfb-11e5-94fa-c8f73332c8f4, 2dd90258-bdfb-11e5-94fa-c8f73332c8f4,
```

Comment:
I changed the original loader to sort metadata key values of list to ensure deterministic results. This is the result of that.